lms/djangoapps/teams/views.py

```diff
@@ -889,6 +888,13 @@ def post(self, request):
         except User.DoesNotExist:
             return Response(status=status.HTTP_404_NOT_FOUND)
 
+        course_module = modulestore().get_course(team.course_id)
+        if course_module.teams_max_size is not None and team.users.count() >= course_module.teams_max_size:
+            return Response(
+                build_api_error(ugettext_noop("The team is already full.")),
```

Comment:
@efischer19 
Only a minor change: "The" -> "This"
"This team is already full."